### PR TITLE
fix(performance) : Fix abend on accessing Performance tab on Non Apache server

### PR DESCRIPTION
**Pull request by Udinra**
If you try to access Performance page from Admin it will abend on Nginx powered Elgg installation as get_apache_version function called in the file will not be defined

### CODE REVIEW

File: views/default/admin/performance/generic.php [hunk 1]
```diff
@@ -26,21 +26,23 @@
 };
 
 // apache version
-if (apache_get_version() !== false) {
-	$icon =  $icon_warning;
-	$title = elgg_echo('admin:performance:apache:mod_cache');
-	$value = elgg_echo('status:unavailable');
-	$subtext = '';
+// Check if the Server is Apache before calling apache_get_version else it may error out in server like Nginx
+if (strpos($_SERVER['SERVER_SOFTWARE'],'Apache') !== false) {
```
Comment (jeabakker) [inline views/default/admin/performance/generic.php]: why not check on the existence of the function? Your way is (currently) case sensitive, and maybe some other webserver says it's Apache when it isn't (can happen with opensource software)

**Review by jeabakker — CHANGES_REQUESTED** [review]
Thanks for the fix, maybe look at my comment.

Also look at the error's reported by Travis (coding style and commit message format) 

http://learn.elgg.org/en/stable/contribute/code.html#commit-message-format

File: views/default/admin/performance/generic.php (hunk 1 above)
Comment (Udinra) [reply views/default/admin/performance/generic.php]: Yeah good suggestion. I will update the code and see Elgg guidelines. That was quick dirty fix to make it working on my personal site

Comment (jeabakker) [reply views/default/admin/performance/generic.php]: > That was quick dirty fix to make it working on my personal site

understand completely ;) now try to do it the 'right' way ;)

**Comment by jeabakker** [comment]
please have a look at the lint errors https://travis-ci.org/Elgg/Elgg/jobs/576382512

**Comment by jeabakker** [comment]
replaced by #12803

**Pull request closed without merging**